.circleci/config.yml

```diff
@@ -183,14 +183,26 @@ commands:
           else
             head -n $part .head_tests.txt > .current_tests.txt
           fi
+          excluded_tests=(
+            "Libplanet.Net.Tests.Transports.NetMQTransportTest.AsPeer"
+            "Libplanet.Tests.Blocks.BlockMetadataTest.CancelMineNonce"
+          )
           cat .current_tests.txt
           first=1
           while read test; do
-            if [[ "$first" = "1" ]]; then
-              echo "FullyQualifiedName=$test"
-              first=0
-            else
-              echo "| FullyQualifiedName=$test"
+            write=1
+            for excl in "${excluded_tests[@]}"; do
+              if [[ "$test" == "$excl"* ]]; then
+                write=0
+              fi
+            done
+            if [[ "$write" == "1" ]]; then
```

Comment:
To give readers more hint at its intention, how about renaming it `write` to `to_write`?  The current name can be interpreted in both ways, *to write* or *written*, which are opposite.

Reply:
Thanks, looks much better.